advanced/Scripts/update.sh

```diff
@@ -24,9 +24,11 @@ CHECK_ONLY=false
 
 # shellcheck disable=SC1090
 source "${PI_HOLE_FILES_DIR}/automated install/basic-install.sh"
-# shellcheck disable=SC1091
-source "/opt/pihole/COL_TABLE"
+# shellcheck source=SCRIPTDIR/COL_TABLE
+source "${PI_HOLE_INSTALL_DIR}/COL_TABLE"
```

Comment:
After rebasing, the issue with the Stickler CI still applies. I do not know if and how I could allow shellcheck from Stickler CI to access the COL_TABLE file. I could add an `shellcheck disable=` to the line but in my opinion it would be better to allow shellcheck accessing COL_TABLE. It would be great if one of the core devs could evaluate this.

Reply:
Just use the path that is valid in the repository, i.e.,
```
# shellcheck source=COL_TABLE
```
if this works (I haven't tested this)